lib/tapjoy/ldap/group/delete.rb

```diff
@@ -0,0 +1,38 @@
+module Tapjoy
+  module LDAP
+    module Group
+      # Delete LDAP group
+      class Delete
+        def delete
+          unless opts[:force] confirm
```

Comment:
Getting this stacktrace when I run ldaptools `/Users/mwalser/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/ldap_tools-0.4.0/lib/tapjoy/ldap/group.rb:2:in `require_relative': /Users/mwalser/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/ldap_tools-0.4.0/lib/tapjoy/ldap/group/delete.rb:7: syntax error, unexpected tIDENTIFIER, expecting keyword_then or ';' or '\n' (SyntaxError)`, referencing this line.